gr-blocks/lib/wavfile_sink_impl.h

```diff
@@ -29,6 +32,7 @@ class wavfile_sink_impl : public wavfile_sink
     float d_min_sample_val;
     float d_normalize_shift;
     float d_normalize_fac;
+    std::vector<float> sample;
```

Comment:
Member variables should have prefix `d_`, so `d_sample`.

This is also now more of a buffer than "a sample", so `d_buf`, or `d_scratch` or something maybe?